ale_linters/go/gobuild.vim

```diff
@@ -39,15 +40,21 @@ function! ale_linters#go#gobuild#GetMatches(lines) abort
 endfunction
 
 function! ale_linters#go#gobuild#Handler(buffer, lines) abort
+    let l:dir = expand('#' . a:buffer . ':p:h')
     let l:output = []
 
     for l:match in ale_linters#go#gobuild#GetMatches(a:lines)
-        " Omit errors from imported go packages
-        if !ale#path#IsBufferPath(a:buffer, l:match[1])
+        "
+        " Get the list of parent paths from the current filename
+        let l:paths = ale#path#Upwards(expand('#' . a:buffer . ':p'))
+        " Only see errors from this package (directory), ignore errors from imported packages
```

Comment:
Why ignore them? You can include them in the loclist so you can go and fix errors in other packages. The only reason other files are ignored at the moment is because ALE didn't support adding those errors to the loclist.

Reply:
Ok